projects/ion/src/lib/message/message.component.scss

```diff
@@ -0,0 +1,43 @@
+@import '../../styles/index.scss';
+@mixin icons-details($color) {
+  ::ng-deep svg {
+    fill: $color;
+  }
+}
+
+.ion-message {
+  display: flex;
+  gap: 16px;
+  box-shadow: 0px 8px 6px -4px rgb(0 0 0 / 15%), 0px 0px 2px rgb(0 0 0 / 15%);
+  height: 40px;
+  width: 100%;
+  font-size: 14px;
+  color: $neutral-7;
+  align-items: center;
+  border-radius: 8px;
+  cursor: pointer;
+
+  &.random {
+    @include icons-details($neutral-6);
+  }
+
+  &.positive {
+    @include icons-details($positive-6);
+  }
+
+  &.negative1 {
+    @include icons-details($negative-6);
+  }
+
+  &.negative2 {
+    @include icons-details($negative-6);
+  }
```

Comment:
O negative 1 e 2 tem as mesmas cores? se for, acredito que seja melhor usar apenas um.
Caso seja diferente, uma sugestão para os nomes poderia ser, erro(x) e alert(!) ou negative_erro(x) e negative_alert(!).

Reply:
Sim, os dois tem a mesma cor.
Vou mudar os nomes.